.github/workflows/stop-dev-environment.yml

```
@@ -0,0 +1,33 @@
+        with:
+          fetch-depth: 0
+      
+      - name: Configure Twingate
+        uses: twingate/github-action@v1
+        with:
+          service-key: ${{ secrets.TWINGATE_SERVICE_KEY_SECRET_NAME }}
+
+      - name: Execute EKS Stop Action
+        uses: ./.github/actions/AWS/stop-dev
+        with:
+          aws-region: ${{ secrets.AWS_REGION }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)